inttest/Makefile

```diff
@@ -19,6 +19,7 @@ bin:
 bin/sonobuoy: | bin
 	$(curl) $(sonobuoy_url) | tar -C bin/ -zxv $(notdir $@)
 
+helm_version = $(strip $(subst v,,$(shell cd .. && go list -m -f '{{.Version}}' helm.sh/helm/v3 2>/dev/null)))
```

Comment:
Hmm, usually this is handled via vars.sh, but that's in the repo root and not part of the inttest folder ... :thinking: But since we have a `cd ..` in here anyways, I'd prefer sth. like `../vars.sh FROM=.. helm_version`.

Or, since we're already looking into Renovate, we could just leave the version here around and add a regex pattern that bundles the updates :thinking:

Reply:
That was my initial intention but I think this is saner.

Anyway I changed it to use `vars.sh`